src/components/course/tests/LicenseRequestedAlert.test.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { render, fireEvent, waitFor } from '@testing-library/react';
+import Cookies from 'universal-cookie';
+import LicenseRequestedAlert from '../LicenseRequestedAlert';
+import { CourseContext } from '../CourseContextProvider';
+import { SubsidyRequestsContext } from '../../enterprise-subsidy-requests';
+import { UserSubsidyContext } from '../../enterprise-user-subsidy';
+import { SUBSIDY_REQUEST_STATE } from '../../enterprise-subsidy-requests/constants';
+import { LICENSE_REQUESTED_ALERT_DISMISSED_COOKIE_NAME } from '../data/constants';
+
+const mockCatalogUUID = 'uuid';
+jest.mock('universal-cookie');
+
+/* eslint-disable react/prop-types */
+const LicenseRequestedAlertWrapper = ({
+  subscriptions = [
+    {
+      enterpriseCatalogUuid: mockCatalogUUID,
+    },
+  ], licenseRequests = [
+    {
+      state: SUBSIDY_REQUEST_STATE.REQUESTED,
+    },
+  ],
+}) => (
+  <UserSubsidyContext.Provider value={{
+    offers: {
+      offers: [],
+      offersCount: 0,
+    },
+    subscriptionLicense: {},
+    customerAgreementConfig: {
+      subscriptions,
+    },
+  }}
+  >
+    <SubsidyRequestsContext.Provider value={
+      {
+        subsidyRequestConfiguration: null,
+        licenseRequests,
+        couponCodeRequests: [],
+        isLoading: false,
+      }
+    }
+    >
+      <CourseContext.Provider>
+        <LicenseRequestedAlert catalogList={[mockCatalogUUID]} />
+      </CourseContext.Provider>
+    </SubsidyRequestsContext.Provider>
+  </UserSubsidyContext.Provider>
+);
+/* eslint-enable react/prop-types */
+
+describe('<LicenseRequestedAlert />', () => {
+  it('renders correctly', () => {
+    const { getByText } = render(<LicenseRequestedAlertWrapper />);
+    expect(getByText('Course requested'));
+    expect(getByText(
+      'Your organization’s subscription covers all of the courses in this catalog.'
+        + ' You have already requested access to all courses.',
```

Comment:
nit: Worth exporting this sentence as a variable that can be re-used in both the component and the test? That way, copy changes in the component won't break tests.